examples/solid/blog/vite.config.ts

```
@@ -1,21 +1,19 @@
-import { defineConfig } from 'vite'
+import { defineConfig, resolveConfig } from 'vite'
```

ESLint

[error] 'resolveConfig' is defined but never used.

(unused-imports/no-unused-imports)